lms/models/lti_registration.py

```diff
@@ -38,3 +38,13 @@ class LTIRegistration(CreatedUpdatedMixin, Base):
     application_instances = sa.orm.relationship(
         "ApplicationInstance", back_populates="lti_registration"
     )
+
+    @property
+    def product_family(self) -> str:
+        """To which LMS (Canvas, D2L, BB..) does this registration belong."""
+        from lms.product.family import Family  # noqa: PLC0415
+
+        if self.issuer.endswith(".instructure.com"):
+            return Family.CANVAS
```

Comment:
What if someone is self-hosting Canvas, or using some other provider? Will that still match that suffix?

Reply:
I'm not 100% sure but I'd say:

- This is a authorization parameter identifying Canvas, it might be the same (ie is hard coded in the code base?) even if self hosting.
- Looks like self hosting Canvas is possible but not common: https://hypothes-is.slack.com/archives/C2BLQDKHA/p1727271613731069#

I reckon we might have to live with this approach and revisit if we encounter one customer where this becomes a problem.

Reply:
> Looks like self hosting Canvas is possible but not common: https://hypothes-is.slack.com/archives/C2BLQDKHA/p1727271613731069#

Yeah, I would be more worried about people using other third party hosting providers which are not the "official" one.

> I reckon we might have to live with this approach and revisit if we encounter one customer where this becomes a problem.

Agree, let's go with this for now.